app/models/manageiq/providers/vmware/cloud_manager.rb

```diff
@@ -102,4 +102,14 @@ def vm_create_snapshot(vm, options = {})
       service.process_task(response.body)
     end
   end
+
+  def vm_remove_snapshot(vm, options = {})
```

Comment:
Please add underscore `_options = {}` because this attribute is not really used by us.